net/cifsd-tools/Makefile

```diff
@@ -18,28 +18,69 @@ PKG_INSTALL:=1
 PKG_BUILD_PARALLEL:=1
 PKG_REMOVE_FILES:=autogen.sh
 
+PKG_BUILD_DEPENDS:=glib2
+
 include $(INCLUDE_DIR)/package.mk
 include $(INCLUDE_DIR)/nls.mk
 
-define Package/cifsd-tools
+define Package/cifsd-tools/Default
   SECTION:=net
   CATEGORY:=Network
   SUBMENU:=Filesystem
-  TITLE:=Kernel CIFS/SMB server support and userspace tools
+  TITLE:=Kernel SMB/CIFS
   URL:=https://github.com/cifsd-team/cifsd-tools
-  DEPENDS:=+kmod-fs-cifsd +glib2 +libnl-core +libnl-genl
+  DEPENDS:= +libnl-core +libnl-genl $(ICONV_DEPENDS) $(INTL_DEPENDS)
 endef
 
-define Package/cifsd-tools/description
-  Userspace tools (cifsd, cifsuseradd, cifsshareadd) for the CIFS/SMB kernel fileserver.
+define Package/cifsd-tools/Default/description
+  Userspace tools for the SMB/CIFS kernel fileserver (cifsd.ko).
   The config file location is /etc/cifs/smb.conf
 endef
 
-define Package/cifsd-tools/install
-	$(INSTALL_DIR) $(1)/usr/lib
-	$(CP) $(PKG_INSTALL_DIR)/usr/lib/libcifsdtools.so* $(1)/usr/lib/
+define Package/cifsd-server
+  $(call Package/cifsd-tools/Default)
+  TITLE+= server
+  DEPENDS+= +kmod-fs-cifsd
+endef
+
+define Package/cifsd-server/description
+  installs: cifsd
+
+  This provides the basic fileserver service and is the minimum needed to serve 'guest only' file shares or use a existing cifsdpwd.db.
+endef
+
+define Package/cifsd-utils
+  $(call Package/cifsd-tools/Default)
+  TITLE+= user management-util
+endef
+
+define Package/cifsd-utils/description
+  installs: cifsuseradd (cifsshareadd)
+
+  Tool needed to create the cifsdpwd.db, to manage per user share passwords.
+  NOTE: Not needed for 'guest only' shares.
+endef
+
+define Package/cifsd-utils/config
+	config CIFSD_UTILS_SHAREADD
+		bool "Add cifsshareadd util"
+		depends on PACKAGE_cifsd-utils
+		help
+			Add the cifsshareadd tool, to directly manipulate the /etc/cifs/smb.conf.
+		default n
+endef
+
+CONFIGURE_ARGS += \
+	--disable-shared \
+	--enable-static \
+	GLIB_LIBS="$(STAGING_DIR)/usr/lib/libglib-2.0.a"
```

Comment:
I think it should be in `CONFIGURE_VARS`. Does it work here too?

Reply:
Yes worked, but you are correct, fixed.